src/widgets/imgbtn/lv_imgbtn.h

```diff
@@ -30,16 +30,20 @@ typedef enum {
     _LV_IMGBTN_STATE_NUM,
 } lv_imgbtn_state_t;
 
+typedef struct {
+    const void * img_src;
+    lv_img_header_t header;
+} lv_imgbtn_src_pair_t;
```

Comment:
Why is it called `pair`? What about `lv_imgbtn_src_info_t`?

Reply:
Because I think they come in pairs

Reply:
I think this name is not so flexible. If for whatever reason we need to add a new member there the name of the type needed to be changed too.

Reply:
done